waku/protocol/v2/waku_filter.nim

```diff
@@ -17,6 +17,11 @@ import
 const
   WakuFilterCodec* = "/vac/waku/filter/2.0.0-alpha2"
 
+# TODO: This will be an asymmetric Protocol so we probably need some full node
+# vs light node identification?
+# Or we check this with ContentFilter somehow? If none is set it can be
```

Comment:
Just a quick comment, will look more on Monday: it isn't clear to me that light and full node in a coarse sense is a good distinction. One can imagine a desktop node being a "light" node in some respects and a "full" node in some respects.

What's wrong with sender of ContentFilter being ""light node"" and receiver ""full node""? Why can't this be inferred on an RPC basis?

Reply:
Yeah, it might be enough (was wondering if there are no cases where it is not enough when switching between these modes, say switching from/to using WakuRelay).

Light node is indeed not the best distinction, I just wanted to make it clear now that based on some "settings" different actions need to be taken when e.g. subscribing.